fastddsspy_yaml/src/cpp/YamlReaderConfiguration.cpp

```diff
@@ -117,30 +119,38 @@ void Configuration::load_dds_configuration_(
     // Get optional allowlist
     if (YamlReader::is_tag_present(yml, ALLOWLIST_TAG))
     {
-        allowlist = YamlReader::get_set<utils::Heritable<IFilterTopic>>(yml, ALLOWLIST_TAG,
+        ddspipe_configuration.allowlist = YamlReader::get_set<utils::Heritable<IFilterTopic>>(yml, ALLOWLIST_TAG,
                         version);
 
         // Add to allowlist always the type object topic
         WildcardDdsFilterTopic internal_topic;
         internal_topic.topic_name.set_value(TYPE_OBJECT_TOPIC_NAME);
-        allowlist.insert(
+        ddspipe_configuration.allowlist.insert(
             utils::Heritable<WildcardDdsFilterTopic>::make_heritable(internal_topic));
     }
 
     /////
     // Get optional blocklist
     if (YamlReader::is_tag_present(yml, BLOCKLIST_TAG))
     {
-        blocklist = YamlReader::get_set<utils::Heritable<IFilterTopic>>(yml, BLOCKLIST_TAG,
+        ddspipe_configuration.blocklist = YamlReader::get_set<utils::Heritable<IFilterTopic>>(yml, BLOCKLIST_TAG,
                         version);
     }
 
+    /////
+    // Get optional topics
+    if (YamlReader::is_tag_present(yml, TOPICS_TAG))
+    {
+        const auto& manual_topics = YamlReader::get_list<ManualTopic>(yml, TOPICS_TAG, version);
+        ddspipe_configuration.manual_topics =
+                std::vector<ManualTopic>(manual_topics.begin(), manual_topics.end());
+    }
+
     /////
     // Get optional builtin topics
     if (YamlReader::is_tag_present(yml, BUILTIN_TAG))
     {
-        // WARNING: Parse builtin topics AFTER specs, as some topic-specific default values are set there
```

Comment:
Is this not the case anymore? If so, I would keep the warning for developers.

Reply:
It isn't the case anymore. There are no longer built-in topics.

Reply:
Well, now I meant for manual topics.

Reply:
The parsing order is irrelevant. The Manual Topics get applied when creating the entities in the DdsBridge, and the Specs Topic QoS get applied on the spot, as they are parsed. In any case, the order can't matter since now the Topic QoS are Fuzzy and the Specs Topic QoS and the Manual Topic QoS have different levels of fuzziness.